internal/command/show.go

```diff
@@ -89,6 +89,19 @@ func mustWriteStringSliceRows(fmt format.Formatter, header string, indentlvl int
 	}
 }
 
+func (*showCmd) strCmd(cmd []string) string {
+	var result strings.Builder
+
+	for i, e := range cmd {
+		result.WriteString(fmt.Sprintf("\"%s\"", e))
```

Comment:
Shouldn't we escape `"` on `e` here? Like:
```go
result.WriteString(fmt.Sprintf("\"%s\"", strings.Replace(e, `"`, `\"`)))
```

Also, it might make sense to include tests with space and the `"` chars.